mslib/mscolab/_tests/test_chat_manager.py

```diff
@@ -24,70 +24,49 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 """
-import os
-import requests
-import json
-import sys
-import pytest
-
-from werkzeug.urls import url_join
-from PyQt5 import QtWidgets, QtTest
-from mslib.mscolab.models import User, MessageType, Message
+from flask import Flask
 from mslib.mscolab.conf import mscolab_settings
+from mslib.mscolab.models import db, User, MessageType, Message
 from mslib.mscolab.chat_manager import ChatManager
-from mslib.msui.mscolab import MSSMscolabWindow
-from mslib._tests.utils import mscolab_start_server
+from mslib.mscolab.seed import add_user, add_project, add_user_to_project, delete_user, delete_project
+from mslib.mscolab.mscolab import handle_db_seed
 
 
-PORTS = list(range(9321, 9340))
-
-
-@pytest.mark.skipif(os.name == "nt",
-                    reason="multiprocessing needs currently start_method fork")
 class Test_Chat_Manager(object):
     def setup(self):
-        self.process, self.url, self.app, _, self.cm, self.fm = mscolab_start_server(PORTS)
-        QtTest.QTest.qWait(500)
-        self.application = QtWidgets.QApplication(sys.argv)
-        self.window = MSSMscolabWindow(data_dir=mscolab_settings.MSCOLAB_DATA_DIR,
-                                       mscolab_server_url=self.url)
+        handle_db_seed()
+        self.app = Flask(__name__, static_url_path='')
+        self.app.config['SQLALCHEMY_DATABASE_URI'] = mscolab_settings.SQLALCHEMY_DB_URI
+        self.app.config['MSCOLAB_DATA_DIR'] = mscolab_settings.MSCOLAB_DATA_DIR
+        self.app.config['UPLOAD_FOLDER'] = mscolab_settings.UPLOAD_FOLDER
+        self.app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+        db.init_app(self.app)
```

Comment:
Factor common starting code out similar to previous mscolab_start_server. Perhaps simply add a new function with a "2"?

Reply:
rephrase this please. Because of my refactoring to test_client this now different.